cmd/aws-lambda-rie/http.go

```diff
@@ -16,15 +21,85 @@ func startHTTPServer(ipport string, sandbox *rapidcore.SandboxBuilder, bs intero
 		Addr: ipport,
 	}
 
+	log.Warnf("Listening on %s", ipport)
+
+	maxInvocations := -1 // -1 means unlimited invocations
+	// Get max invocations from environment variable
+	maxInvocationsStr := os.Getenv("AWS_LAMBDA_SERVER_MAX_INVOCATIONS")
+	if maxInvocationsStr != "" {
+		if maxInvocationsInt, err := strconv.Atoi(maxInvocationsStr); err == nil {
+			maxInvocations = maxInvocationsInt
+		} else {
+			log.Panicf("Invalid value for AWS_LAMBDA_SERVER_MAX_INVOCATIONS: %s", maxInvocationsStr)
+		}
+	}
+
+	// Channel to signal server shutdown
+	shutdownChan := make(chan struct{})
+
 	// Pass a channel
 	http.HandleFunc("/2015-03-31/functions/function/invocations", func(w http.ResponseWriter, r *http.Request) {
-		InvokeHandler(w, r, sandbox.LambdaInvokeAPI(), bs)
+		InvokeHandler(w, r, sandbox.LambdaInvokeAPI(), bs, func(invokeResp *ResponseWriterProxy){
+
+			// Forward response if "forward-response" header exists
+			if forwardURL := r.Header.Get("forward-response"); forwardURL != "" {
+				// Create a wait group to wait for the API request to finish
+				var wg sync.WaitGroup
+				wg.Add(1)
+				go func() {
+					// Marshal the payload to JSON (you can use any other serialization format if needed)
+					apiPayloadJSON, err := json.Marshal(string(invokeResp.Body))
+					if err != nil {
+						log.Errorf("Failed to json marshal API payload: %s", err)
+						return
+					}
+
+					// Create an API request to the URL in the "forward-response" header
+					client := &http.Client{}
+					req, err := http.NewRequest("POST", forwardURL, bytes.NewReader(apiPayloadJSON))
```

Comment:
if `forward-response` exists in the headers, send a POST request to the URL at the `forward-response`